configs/sst_storage_io-packages-unwanted.yaml

```diff
@@ -6,8 +6,10 @@ data:
   maintainer: sst_storage_io
 
   unwanted_packages:
+  - ktls-utils
   - libhbaapi
   - libhbalinux
+  - nvme-stas
```

Comment:
This is provided in CentOS Stream/RHEL 9 today and isn't going to be removed at this point.  It would be better to create an `eln` specific file and include it there.

Reply:
Ah, that makes sense.  I will split the eln and c9s labels and only add these packages to that.  I wasn't sure if existing c9s information was still relevant.  Thank you for confirming.